scripts/postgenerate-orchestration.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
+import { resolve } from 'node:path';
+import { transformFilesInDirectory } from './util.js';
+
+const correctedNames = [
+  ['LLMModuleResult', 'LlmModuleResult'],
```

Comment:
[idea] maybe we can add a util function in cloud sdk with v.capitalize(v.camelCase(str)) to do this.